lib/gooddata/bricks/middleware/gooddata_middleware.rb

```diff
@@ -48,7 +52,8 @@ def call(params)
           params['GDC_VERIFY_SSL'].to_b,
           params['GDC_USERNAME'],
           params['GDC_PASSWORD'],
-          params['GDC_SST']
+          params['GDC_SST'],
```

Comment:
info: Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy) (reek)